mtest/libmscore/instrumentchange/tst_instrumentchange.cpp

```diff
@@ -95,7 +96,9 @@ void TestInstrumentChange::testDelete()
       Measure* m = score->firstMeasure()->nextMeasure();
       Segment* s = m->first(SegmentType::ChordRest);
       InstrumentChange* ic = toInstrumentChange(s->annotations()[0]);
+	score()->startCmd();
       score->deleteItem(ic);
+	score()->endCmd();
```

Comment:
There's some weird indentation here. Also, tests are failing because `score()` doesn't exist in this scope. I think you mean `score->startCmd()` (without parentheses), because `score` is assigned a few lines above.